api/controllers/swagger.go

```diff
@@ -423,6 +429,51 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 					},
 				},
 			},
+			"/tag": map[string]swagger.Method{
+				"delete": {
```

Comment:
[nit] Other entities in the swagger controller list List first, then Add, then Update (if applicable), then Delete